src/app/features/notification/notification.component.ts

```diff
@@ -0,0 +1,113 @@
+import { Component, OnInit } from "@angular/core";
+import { MatBadgeModule } from "@angular/material/badge";
+import { FontAwesomeModule } from "@fortawesome/angular-fontawesome";
+import { MatMenu } from "@angular/material/menu";
+import { MatButtonModule } from "@angular/material/button";
+import { MatMenuTrigger } from "@angular/material/menu";
+import { MatMenuModule } from "@angular/material/menu";
+import { FormsModule } from "@angular/forms";
+import { MatTooltipModule } from "@angular/material/tooltip";
+import { Logging } from "app/core/logging/logging.service";
+import { NotificationEvent } from "./model/notification-event";
+import { EntityMapperService } from "app/core/entity/entity-mapper/entity-mapper.service";
+import { MatTabsModule } from "@angular/material/tabs";
+import { NotificationItemComponent } from "./notification-item/notification-item.component";
+import { MockNotificationsService } from "./mock-notification.service";
+import { SessionSubject } from "app/core/session/auth/session-info";
+
+@Component({
+  selector: "app-notification",
+  standalone: true,
+  imports: [
+    MatBadgeModule,
+    FontAwesomeModule,
+    MatMenu,
+    MatButtonModule,
+    MatMenuTrigger,
+    MatMenuModule,
+    FormsModule,
+    MatTooltipModule,
+    MatTabsModule,
+    NotificationItemComponent,
+  ],
+  templateUrl: "./notification.component.html",
+  styleUrl: "./notification.component.scss",
+})
+export class NotificationComponent implements OnInit {
+  public allNotifications: NotificationEvent[] = [];
+  public unreadNotifications: NotificationEvent[] = [];
+  public selectedTab = 0;
+
+  constructor(
+    private entityMapper: EntityMapperService,
+    private mockNotificationsService: MockNotificationsService,
+    private sessionInfo: SessionSubject,
+  ) {}
+
+  ngOnInit(): void {
+    this.loadAndProcessNotifications();
+  }
+
+  /**
+   * Loads all notifications and processes them to update the list and unread count.
+   */
+  private async loadAndProcessNotifications(): Promise<void> {
+    // TODO: Need to uncomment this after the notification list UI testing is completed.
+    // const notifications =
+    // await this.entityMapper.loadType<NotificationEvent>(NotificationEvent);
+
+    // TODO: Need to remove this line once the notification list UI tests are completed.
+    const notifications =
+      this.mockNotificationsService.getNotifications() as unknown as NotificationEvent[];
+
+    // The user is hardcoded for testing purposes, need to remove this.
+    this.filterUserNotifications(
+      notifications,
+      this.sessionInfo.value?.entityId,
```

Comment:
needs to be id, not entityId (which again may not be defined for every user account)
```suggestion
    this.filterUserNotifications(
      notifications,
      this.sessionInfo.value?.id,
```